libs/api-client-tiger/src/index.ts

```diff
@@ -53,11 +58,21 @@ export {
 };
 
 export interface ITigerClient {
+    axios: AxiosInstance;
```

Comment:
This is scary. I understand that this is a temporary solution, be we should probably mark this as deprecated so that nobody uses this and remove this implementation detail leak as soon as possible.

Reply:
what's wrong with exposing the direct API client instance?

Reply:
i mean.. we also expose newAxios factory and is already marked as public in api-extractor

Reply:
Oh, I was not aware of that :-/ I mean it ties the API with a concrete implementation, but since it is tied already, this is non issue